core/vm/lfvm/interpreter.go

```diff
@@ -657,7 +676,8 @@ func steps(c *context, one_step_only bool) {
 		case NOOP:
 			opNoop(c)
 		case DATA:
-			panic("can not interpret data as instruction")
+			c.status = INVALID_INSTRUCTION
```

Comment:
Should we have a dedicated status code for this? The analog in other contexts would be a "segmentation fault". Maybe "INVALID_JUMP"?

Reply:
If interpreter gets to this point and should process DATA instruction itself, which should not occur, then INVALID_INSTRUCTION status was translated into vm.ErrInvalidCode in interpreter. But I agree, it may have separate status. I'll add SEGMENTATION_FAULT status